addon/components/fl-select.ts

```diff
@@ -40,6 +41,11 @@ export default class FlSelect extends FlInput<FlSelectArgs> {
         return !this.allowClear;
     }
 
+    get selectName() {
```

Comment:
id probably rename the getter in `FlInput` to something more generic like just `get name()`, so that you don't have to reimplement in here or FlTextarea, since both extend `FlInput`.